waspc/cli/src/Wasp/Cli/Command/Build.hs

```diff
@@ -37,12 +34,18 @@ build = do
   let buildDir =
         waspProjectDir </> dotWaspDirInWaspProjectDir
           </> buildDirInDotWaspDir
-      buildDirFilePath = SP.fromAbsDir buildDir
 
-  doesBuildDirExist <- liftIO $ doesDirectoryExist buildDirFilePath
+  doesBuildDirExist <- liftIO $ doesDirectoryExist buildDir
   when doesBuildDirExist $ do
     cliSendMessageC $ Msg.Start "Clearing the content of the .wasp/build directory..."
-    liftIO $ removeDirectoryRecursive buildDirFilePath
+    liftIO $ removeDirectory buildDir
+    cliSendMessageC $ Msg.Success "Successfully cleared the contents of the .wasp/build directory."
+
+  let sdkDir = waspProjectDir </> dotWaspDirInWaspProjectDir </> generatedCodeDirInDotWaspDir </> sdkRootDirInProjectRootDir
+  doesSdkDirExist <- liftIO $ doesDirectoryExist sdkDir
+  when doesSdkDirExist $ do
+    cliSendMessageC $ Msg.Start "Clearing the content of the .wasp/build directory..."
```

Comment:
Is this message correct since you are deleting the `sdk` dir, and not the whole `build` dir?